js/gameConfig.js

```diff
@@ -34,20 +36,20 @@ const skillBaseData = {
-    Homeless: {title: 'Homeless', expense: 0, effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 5}]},
-    Book: {title: 'Book', expense: 10, effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 5}], description: 'Research xp'},
+    Homeless: {title: 'Homeless', expense: 0, effects: [{effectType: EffectType.Population, baseValue: 5}]},
+    Book: {title: 'Book', expense: 10, effects: [{effectType: EffectType.Population, baseValue: 5}], description: 'Research xp'},
 };
 
 const moduleOperations = {
-    Garbage: new ModuleOperation({title: 'Garbage', effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 5}], maxXp: 400}),
-    Diesel: new ModuleOperation({title: 'Diesel', effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 5}], maxXp: 50}),
-    Plastics: new ModuleOperation({title: 'Plastics', effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 5}], maxXp: 100}),
-    Steel: new ModuleOperation({title: 'Steel', effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 5}], maxXp: 200}),
-    QuantumReplicator: new ModuleOperation({title: 'Quantum Replicator', effects: [{effectType: EffectType.Population, baseValue: 5}], maxXp: 400, energyConsumption: 30}),
-    BioGenesisChamber: new ModuleOperation({title: 'Bio-Genesis Chamber', effects: [{effectType: EffectType.Population, baseValue: 5}], maxXp: 400, energyConsumption: 40}),
-    NanoFertilityDrones: new ModuleOperation({title: 'Nano-Fertility Drones', effects: [{effectType: EffectType.Population, baseValue: 5}], maxXp: 400, energyConsumption: 50}),
-    HoloCommunityHub: new ModuleOperation({title: 'Holo-Community Hub', effects: [{effectType: EffectType.Population, baseValue: 5}], maxXp: 400, energyConsumption: 60}),
-    TemporalBreedingPods: new ModuleOperation({title: 'Temporal Breeding Pods', effects: [{effectType: EffectType.Population, baseValue: 5}], maxXp: 400, energyConsumption: 80}),
+    Garbage: new ModuleOperation({title: 'Garbage', effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 1}], maxXp: 400, energyConsumption: 1}),
+    Diesel: new ModuleOperation({title: 'Diesel', effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 2}], maxXp: 50, energyConsumption: 1}),
+    Plastics: new ModuleOperation({title: 'Plastics', effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 1}], maxXp: 100, energyConsumption: 1}),
+    Steel: new ModuleOperation({title: 'Steel', effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.EnergyFactor, baseValue: 2}], maxXp: 200, energyConsumption: 1}),
```

Comment:
EnergyFactor hier macht nen guten Eindruck 👍 
ich würde allerdings gerne noch `energyConsumption` umbenannt sehen in `gridLoad`